tests/unit/cartography/intel/github/test_teams.py

```diff
@@ -123,8 +123,8 @@ def test_get_team_users_happy_path(mock_get_team_users):
 
 
 @patch('cartography.intel.github.teams._get_team_repos')
-@patch('cartography.intel.github.teams.sleep')
-def test_get_team_repos_github_returns_none(mock_sleep, mock_get_team_repos):
+@patch('cartography.intel.github.teams.backoff_handler', spec=True)
```

Comment:
I didn't _need_ to replace 'sleep' with 'backoff_handler', we could've had a fine test without it.  But, it did give a nice extra affirmation that the decorator is wired in correctly, because we can tell that it does its backoff handling the expected number of times.

The `spec=True` was needed, if I understand right, to fool the decorator into thinking it had a backoff handler func it could use.  Without this, with just a MagicMock, it wasn't being invoked.  With this, it was being invoked exactly as expected.